### ISSUE TITLE
NTR: Anti desmocollin-1 IgA in blood (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8792]
New term request Anti desmocollin-1 IgA in blood (segal_220411214135): Keratinocyte antigen

**Comment by pnrobinson** [comment]
@MickeySegal
the HPO terms for autoantibodies do not currently distinguish between immunoglobulin classes. We plan to add modifiers.
Will this work here?

**Comment by MickeySegal** [comment]
We can work with that.  Though I thought that one of the advantages of an ontology is that you can have subtype findings as children of a general finding.

**Comment by pnrobinson** [comment]
I am going to add modifiers for IgA, IgG, IgM positivity, otherwise we will get a combinatorial explosion.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000448
- label: Anti-desmocollin antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:34567003
- def: The presence of autoantibodies (immunoglobulins) in the serum that react against 21-hydroxylase.
- comment: Autoimmune blistering diseases (AIBD) encompass a group of disorders characterized clinically by the development of cutaneous and/or mucosal blisters and erosions. Patients synthesize autoantibodies against structural proteins of epidermis and/or dermal-epidermal junction. Pemphigus is a AIBD skin disease caused by the production of autoantibodies that target desmosomal adhesion molecules including desmoglein and desmocollin, among others.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8792
- synonyms:
